Research/zhaoqix2/flaskr/__init__.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from flask import Flask
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello():
+    return 'Hello, World!'
+
+@app.route('/greeting')
+def greeting():
+    return 'Have a nice day!'
+
+@app.route('/hello/Zhaoqi')
+def helloPerson():
+    return 'Hello Zhaoqi!'
```

Comment:
https://flask.palletsprojects.com/en/1.1.x/quickstart/#variable-rules may help with completing the third part of the tutorial